www/js/app.js

```diff
@@ -1290,17 +1352,32 @@ function handleIframeDrop (e) {
     e.preventDefault();
 }
 
-function handleFileDrop (packet) {
+async function handleFileDrop (packet) {
     packet.stopPropagation();
     packet.preventDefault();
     configDropZone.style.border = '';
     var files = packet.dataTransfer.files;
-    document.getElementById('openLocalFiles').style.display = 'none';
+    document.getElementById('selectInstructions').style.display = 'none';
+    document.getElementById('fileSelectionButtonContainer').style.display = 'none';
     document.getElementById('downloadInstruction').style.display = 'none';
     document.getElementById('selectorsDisplay').style.display = 'inline';
-    setLocalArchiveFromFileList(files);
-    // This clears the display of any previously picked archive in the file selector
     document.getElementById('archiveFiles').value = null;
+
+    const isFSAPIsupported = typeof window.showOpenFilePicker === 'function'
+    const isWebkitSupported = 'webkitdirectory' in document.createElement('input')
```

Comment:
This variable isn't descriptive enough. There are lots of `Webkit` APIs, and it sounds like you're testing whether the browser (Safari) is supported... I think you have to rename to `isWebkitDirAPISupported`.

However, you define these in several places. It would be better to define them and test them once. Since you need them across modules, you could add them as `params` (because `params` is a global object defined in `init.js`, and is available app-wide, including in modules like `abstractFileSystemAccess.js` (where you also need these variables).

The way to define them in `init.js` would be  add `@property {Boolean} isFSAPIsupported` and same for `isWebkitDirAPISupported` in the `AppParams` block in `init.js`. You don't actually need to list them and set them there, though you could do along with other params if you preferred. But it's probably best to set them in `app.js` ideally outside of any function block, and somewhere near the top along with other global defs, maybe after `appstate['pwaUpdateNeeded']`.